src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx

```diff
@@ -122,11 +126,29 @@ function ConfirmationStep({policyID, backTo}: ConfirmationStepProps) {
                         isAlertVisible={!!errorMessage}
                         isDisabled={isOffline}
                         isLoading={issueNewCard?.isLoading}
-                        onSubmit={submit}
+                        onSubmit={() => setIsValidateCodeActionModalVisible(true)}
                         buttonText={translate('workspace.card.issueCard')}
                     />
                 </View>
             </ScrollView>
+            <ValidateCodeActionModal
+                handleSubmitForm={submit}
+                isLoading={issueNewCard?.isLoading}
+                sendValidateCode={() => User.requestValidateCodeAction()}
+                validateError={validateLoginError}
+                clearError={() => {
+                    Card.clearIssueNewCardError();
+                }}
+                onClose={() => {
+                    if (validateLoginError && pendingContactAction?.contactMethod) {
```

Comment:
We dont even need this condition, our error isn't related to login error. Let's remove it as well